docs/guide/basics/sizing.md

```diff
@@ -0,0 +1,87 @@
+<script setup>
+import { Sandpack } from 'sandpack-vue3'
+import sandpackTheme from '../../.vitepress/theme/sandpackTheme.json'
+import sizeIconExample from './examples/size-icon-example/files.ts'
+import sizeIconCssExample from './examples/size-icon-css-example/files.ts'
+import sizeIconFontExample from './examples/size-icon-font-example/files.ts'
+import sizeIconTailwind from './examples/size-icon-tailwind-example/files.ts'
+</script>
+
+# Sizing
+
+By default, the size of all icons is `24px` times `24px`. The size is adjustable using the `size` prop and CSS.
+
+## Adjusting the icon size using the `size` prop
+
+<Sandpack
+  template="react"
+  :theme="sandpackTheme"
+  :files="sizeIconExample"
+  :customSetup='{
+    dependencies: {
+      "lucide-react": "latest"
+    }
+  }'
+  :options="{
+    editorHeight: 300,
+    editorWidthPercentage: 60,
+  }"
+/>
+
+## Adjusting the icon size via CSS
+
+The CSS properties `width` and `height` can be used to adjust the icon size.
+
+<Sandpack
+  template="react"
+  :theme="sandpackTheme"
+  :files="sizeIconCssExample"
+  :customSetup='{
+    dependencies: {
+      "lucide-react": "latest"
+    }
+  }'
+  :options="{
+    editorHeight: 300,
+  }"
+/>
+
+### Dynamically change the icon size based on the font size
+
+It is possible to resize icons based on font size. This can be accomplished by using the `em`s to resize based on font size. See this [MDN article](https://developer.mozilla.org/en-US/docs/Web/CSS/font-size#ems) about ems.
```

Comment:
>This can be accomplished by using the `em`s to resize based on font size.

I would change that to:

>This can be achieved using the `em` unit. See this MDN article for more information on the `em` unit.